db/special_docs.go

```diff
@@ -32,6 +32,10 @@ func (db *DatabaseCollection) GetSpecial(doctype string, docid string) (Body, er
 }
 
 func (db *DatabaseCollection) GetSpecialBytes(doctype string, docid string) ([]byte, error) {
```

Comment:
at this point, i'd change the receiver type to `c` becuase it's not a db operation

Reply:
Done in a single commit but it added a decent chunk of changes:

```
 10 files changed, 142 insertions(+), 142 deletions(-)
```